packages/node_modules/@webex/plugin-meetings/sample/app.js

```diff
@@ -109,6 +112,9 @@ function connect() {
 
   spark.meetings.on('meeting:removed', (m) => {
```

Comment:
Just a reminder we will have to descope these event names and make them consistent with Hydra.

Reply:
I haven't seen anything for that but I'm down to take another look at the events with you matt

Reply:
Thanks, @chrisadubois ! We need to remove the `'meeting:'` prefix first, then ensure the names are consistent with Hydra. This became more apparent to me after integrating Mercury events.